pkg/goversion/go_version.go

```diff
@@ -150,3 +151,28 @@ func Installed() (GoVersion, bool) {
 
 	return Parse(s[len(goVersionPrefix):])
 }
+
+// RuntimeAfterOrEqual checks that the runtime version is major.minor or a
+// later version, or a development version.
+func RuntimeAfterOrEqual(major, minor int) bool {
```

Comment:
Replaced it with VersionAfterOrEqual and passing the version string as argument. We could also move it in pkg/proc/test but I think it's general enough that it can live here.